packages/api-reference/src/standalone/lib/register-globals.ts

```diff
@@ -15,10 +16,8 @@ export const registerGlobals = () => {
     return
   }
 
-  const globalWindow = window
```

Comment:
not sure what that was for  @hanspagel ?

Reply:
Left-over